scripts/vpn-port-guard.sh

```
@@ -1,361 +1,292 @@
-
-    local status status_rc
-    status_rc=0
-    status="$(gluetun_api_status 2>/dev/null || printf 'unknown')" || status_rc=$?
-    local port port_rc
-    port_rc=0
-    port="$(gluetun_api_forwarded_port 2>/dev/null || printf '0')" || port_rc=$?
-
-    local last_error=""
-
-    local forwarded_port="0"
+    last_error=""
```

Shellcheck (0.11.0)

[info] This is actually an end quote, but due to next char it looks suspect.

(SC1079)